tools/marvin/marvin/config/test_data.py

```diff
@@ -775,6 +775,15 @@
         "ostype": "CentOS 5.6 (64-bit)"
 
     },
+    "CentOS6.3template": {
+        "displaytext": "Centos",
+        "name": "Centos",
+        "passwordenabled": False,
+        "ostype": "CentOS 6.3 (64-bit)",
+        "url": "http://10.147.28.7/templates/centos63.ova",
```

Comment:
@bhaisaab this is a simple centos template for vmware so nothing special about it. for this test to pass in one's environment he should have centos template uploaded in their http server and change the url accordingly . 
If you have a place where we can share this file publicly i will be more than happy to upload it there. Not only this file, i will upload  lot other  files  whose urls are a private http server mentioned in test_data.py